src/ToolBox/SOS/NETCore/SOS.NETCore.csproj

```diff
@@ -48,14 +48,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETCore.Platforms">
```

Comment:
I take it we're safe removing this because of https://github.com/dotnet/coreclr/blob/0fd82ac5feaf98d493973a1e1a5de95ba1812943/src/.nuget/init/init.csproj#L10?